src/Components/test/testassets/ComponentsApp.App/Shared/NavMenu.razor

```diff
@@ -37,11 +37,6 @@
                 <span class="oi oi-list-rich" aria-hidden="true"></span> Error
             </NavLink>
         </li>
-        <li class="nav-item px-3">
-            <NavLink class="nav-link" href="links">
-                <span class="oi oi-list-rich" aria-hidden="true"></span> Links
-            </NavLink>
-        </li>
```

Comment:
???

Reply:
The link doesn't exist. When I was trying out my super awesome script to test this: https://github.com/aspnet/AspNetCore/pull/10431#issuecomment-494602536, navigating to this link results in a 404 so I ended up removing it.